web/packages/teleterm/src/services/config/appConfigSchema.ts

```diff
@@ -226,6 +226,10 @@ export const createAppConfigSchema = (settings: RuntimeSettings) => {
       .boolean()
       .default(!settings.dev)
       .describe('Controls whether the hardware key agent will be started.'),
+    'debug.searchResultsScore': z
```

Comment:
It seems to me that this property is quite minor to justify a dedicated entry.
Maybe we could introduce something more generic for the future, like `debug.ui`?